components/mpas-framework/src/framework/mpas_pool_routines.F

```diff
@@ -932,20 +932,32 @@ end subroutine mpas_pool_clone_pool!}}}
 !> copy the data from the members of srcPool into the members of destPool.
 !
 !-----------------------------------------------------------------------
-   recursive subroutine mpas_pool_copy_pool(srcPool, destPool)!{{{
+   recursive subroutine mpas_pool_copy_pool(srcPool, destPool, overrideTimeLevels)!{{{
 
       implicit none
 
       type (mpas_pool_type), pointer :: srcPool
       type (mpas_pool_type), pointer :: destPool
+      integer, intent(in), optional :: overrideTimeLevels
```

Comment:
Would be nice to add a comment on what this is and how it is used.  I realize the rest of framework is not documented, so that's sort of unfair to ask.

Reply:
@matthewhoffman, good point. I added a comment under the `\details` section of the header for the subroutine. Let me know if you have suggestions to improve it.

Reply:
Thanks, @sbrus89 , looks good